src/components/pipeline/EditCreatePipeline.js

```diff
@@ -0,0 +1,270 @@
+import React, { useState } from 'react';
```

Comment:
in this file, would be helpful to add comments on things that are just placeholders. e.g. _this function will be updated when..._ or _todo: update this function to..._ or _mock data to be removed when..._